ci/scripts/r_deps.sh

```diff
@@ -22,6 +22,14 @@ set -ex
 
 source_dir=${1}/r
 
+# Install minimal development dependencies
```

Comment:
@nealrichardson I don't know if you want this. It makes things faster locally but it also means that CI may not check the full staged installation.

Reply:
Yeah I'd prefer not having this on in CI. What if you wrapped it in a check for an env var that you can set locally?

In practice, bison/flex/m4 seem pretty quick to build so it's not a huge deal, is it?

Reply:
Their configure phase is slowish. But, yes, I can remove it. An environment variable sounds like too much hassle.